blockproducer/interfaces/transaction.go

```diff
@@ -59,8 +59,10 @@ const (
 	TransactionTypeAlterDatabaseUser
 	// TransactionTypeDeleteDatabaseUser defines database user deletion transaction type.
 	TransactionTypeDeleteDatabaseUser
-	// TransactionTypeBaseAccount defines base account.
+	// TransactionTypeBaseAccount defines base account transaction type.
 	TransactionTypeBaseAccount
+	// TransactionTypeDeleteDatabaseUser defines database creation transaction type.
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported const TransactionTypeCreataDatabase should be of the form "TransactionTypeCreataDatabase ..."